app/src/main/java/ru/vssemikoz/newsfeed/fragments/newsFeedFragment.java

```diff
@@ -0,0 +1,30 @@
+package ru.vssemikoz.newsfeed.fragments;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import ru.vssemikoz.newsfeed.R;
+
+public class newsFeedFragment extends Fragment {
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
```

Comment:
There's no need in overriding a method without any logic. Let's remove it in case it's not required.